webrender/src/platform/windows/font.rs

```diff
@@ -131,7 +131,10 @@ impl FontContext {
         }
 
         let system_fc = dwrote::FontCollection::system();
-        let font = system_fc.get_font_from_descriptor(&font_handle).unwrap();
+        let font = match system_fc.get_font_from_descriptor(&font_handle) {
```

Comment:
This would be better written as:

```
let font = system_fc
  .get_font_from_descriptor(&font_handle)
  .expect(&format!("missing descriptor {:?}", font_handle));
```

Reply:
Yeah it would

Reply:
Fair point :)

Reply:
Maybe then this ?
```
let font = system_fc
  .get_font_from_descriptor(&font_handle)
  .unwrap_or_else(|| panic!("missing descriptor {:?}", font_handle));
```